pytest_pylint/plugin.py

```diff
@@ -9,10 +9,10 @@
 from configparser import ConfigParser, NoSectionError, NoOptionError
 from os.path import exists, join, dirname
 
-from pylint import lint
-from pylint.config import PYLINTRC
 import pytest
 import toml
+from pylint import lint
```

Comment:
Can you leave these where they are? They're alphabetized. Also, it seems unrelated to the other changes

Reply:
Sure. Done

Reply:
Sorry, this was my fingers automatic doing ctrl+alt+o in pycharm haha